### PR TITLE
html-css/Shahnawaz

### PR DESCRIPTION
**Pull request by shazkhan2**

### CODE REVIEW

File: 1_html-css/CSS_CV.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
+body {
```
Comment (shpomp) [inline 1_html-css/CSS_CV.css:1]: The CSS is not very well formatted. Try to clean it up and/or search on the internet how to automate it with Prettier!

File: 1_html-css/cv.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      <section>
+        <h2 id="contact">Contact</h2>
+        If you need assistance with your projects, please send me an
+        <a href="seewardak@hotmail.com">email </a> with all relevant
+        information. Alternatively, feel free to contact me via:
+        <div id="SoMe">
+          <a href="facebook.com">
+            <img src="./Facebook.png" />
+          </a>
+          <a href="linked.com"> <img src="./Linkedin.png" /> </a>
```
Comment (shpomp) [inline 1_html-css/cv.html:84]: The images are very big in comparison to other elements in the page. Might be a good idea to adjust their displayed size! 💡

**Review by shpomp — APPROVED** [review]
Overall a very good job - nice and simple! 🤩 

You are missing one requirement though:
 - a skill showcase section highlighting your technical skills. 
 
 I also left a few comments for possible appearance/code improvements. However, there was not much to comment on as you did a nice implementation. The HTML especially is well structured, using semantic elements and so on. Good job! 💪🏽

File: 1_html-css/CSS_CV.css (hunk 1 above)
Comment (shazkhan2) [reply 1_html-css/CSS_CV.css:1]: Prettier is on on my CSS as well.
Also, one more section that was missing has been added.

File: 1_html-css/cv.html (hunk 2 above)
Comment (shazkhan2) [reply 1_html-css/cv.html:112]: I have changed the image sizes, as requested.

**Pull request closed without merging**